### ISSUE TITLE
How can I do not use the pretraining model?

### ISSUE THREAD

**Issue by hustCYQ** [open-mmlab/mmdetection#7817]
I use resnet50_fpn, But I do not want to use pretraining model, so I let **model = dict( backbone=dict( init_cfg=None))**. But I found that the default parameters, importing the pretraining model or setting  init_cfg to none all have the same loss, which is around 1.0. 

I found that the loss in the official readme finally converged to around 0.5, so it's not that I canceled the pretraining model, but that the pretraining model was not loaded at all?

**Comment by PeterVennerstrom** [comment]
The model.backbone.init_cfg setting only loads weights from the pretrained backbone.  Typically, neck and head layers for a detection task are added to the backbone and trained on a detection dataset like COCO.  The neck and head layers are initialized with various methods and the backbone is usually initialized with pretrained weights.   

When only loading the pretrained backbone weights, the neck and head weights are untrained so it's possible loss would be fairly similar with or without pretrained backbone weights.

To load the full COCO trained network backbone/neck/head:

`load_from='path/to/weights.pth` 

Needs to be added to the config.

**Comment by hustCYQ** [comment]
After I import the pretraining model, the log shows the unexpected key. Does this mean that the backbone was not imported correctly? In addition, model.backbone. init_cfg mean?
![image](https://user-images.githubusercontent.com/39451786/165005817-bf260d9e-ab74-4e1e-a13e-348f8d3137ef.png)

**Comment by PeterVennerstrom** [comment]
Looks like the full model weights, backbone/neck/head were loaded into the backbone with the init_cfg.  The init_cfg is intended only to load pretrained backbone weights.  Backbone only weights do not include the 'backbone' prefix.  Please see the [default settings](https://github.com/open-mmlab/mmdetection/blob/c72bc707e661d61cf09aca0a53ad21812ef874d0/configs/_base_/models/fast_rcnn_r50_fpn.py#L13). 

The backbone weights will be overwritten when using load_from or resume_from.

**Comment by Petros626** [comment]
@PeterVennerstrom hey, did you worked with `mmrotate` and can tell me, how to train from scratch (no weights for backbone)?
